iocage_lib/listable_resources.py

```diff
@@ -0,0 +1,68 @@
+import collections.abc
+import os
+import re
+import requests
+
+from iocage_lib.ioc_fetch import IOCFetch
+from iocage_lib.ioc_json import IOCZFS
+from iocage_lib.release import Release
+
+
+class ListableResource(collections.abc.Iterable):
```

Comment:
And this should be in resource.py? Why separate them

Reply:
Well my thinking was having all resources which can be listed in a separate file. If you think it makes more sense to have these in `resource.py`, i'll update accordingly. ( i didn't want to populate a single namespace too much )

Reply:
Its easier to use if they are in the same, they are all about the same thing, resources (listing and the resource itself).